app/models/search.rb

```diff
@@ -0,0 +1,10 @@
+class Search < ActiveRecord::Base
+  def self.search(search)
```

Comment:
This should be on the items model. It should be a method you call on the Items class since you are searching for Items.